boxoffice/views/admin_report.py

```diff
@@ -49,19 +48,88 @@ def row_handler(row):
     (ItemCollection, {'id': 'ic_id'}, 'item_collection'),
     permission='org_admin')
 def attendees_report(item_collection):
-    headers = ['receipt_no', 'ticket_no', 'ticket_id', 'ticket_type', 'attendee_fullname', 'attendee_email', 'attendee_phone']
+
+    attendee_details_headers = []
+    for item in item_collection.items:
+        if item.assignee_details:
+            for detail in item.assignee_details.keys():
+                attendee_detail_prefexed = 'attendee_details_'+detail
+                if attendee_detail_prefexed not in attendee_details_headers:
+                    attendee_details_headers.append(attendee_detail_prefexed)
+
+    headers, rows = item_collection.fetch_assignee_details()
+    headers.extend(attendee_details_headers)
+
+    if 'attendee_details' in headers:
+        attendee_details_index = headers.index('attendee_details')
+    else:
+        attendee_details_index = -1
+
+    def row_handler(row):
+        # Convert row to a dict
+        dict_row = {}
+        for idx, item in enumerate(row):
+            # 'assignee_details' is a dict already, so copy and include prefixs
+            if idx == attendee_details_index and isinstance(item, dict):
+                for key in item.keys():
+                    dict_row['attendee_details_'+key] = item[key]
+            # Value is a string, add it to the dict with the corresponding key
+            else:
+                dict_row[headers[idx]] = item
+        # Localize datetime
+        if dict_row.get('date'):
```

Comment:
Use `if 'date' in dict_row`. But you shouldn't be hardcoding for a specific header.